doc/rados/operations/crush-map.rst

```diff
@@ -150,41 +149,42 @@ used to describe these nodes.  Default types include:
 - ``region``
 - ``root``
 
-Most clusters use only a handful of these types, and others
-can be defined as needed.
+Most clusters use only a handful of these types, and other types can be defined
+as needed.
+
+The hierarchy is built with devices (normally of type ``osd``) at the leaves
+and non-device types as the internal nodes. The root node is of type ``root``.
+For example:
 
-The hierarchy is built with devices (normally type ``osd``) at the
-leaves, interior nodes with non-device types, and a root node of type
-``root``.  For example,
 
 .. ditaa::
 
-                        +-----------------+
+                        +-----------------+ 
                         |{o}root default  |
-                        +--------+--------+
+                        +--------+--------+     
                                  |
                  +---------------+---------------+
                  |                               |
           +------+------+                 +------+------+
-          |{o}host foo  |                 |{o}host bar  |
+          |{o}host foo  |                 |{o}host bar  | 
           +------+------+                 +------+------+
                  |                               |
          +-------+-------+               +-------+-------+
          |               |               |               |
    +-----+-----+   +-----+-----+   +-----+-----+   +-----+-----+
-   |   osd.0   |   |   osd.1   |   |   osd.2   |   |   osd.3   |
+   |   osd.0   |   |   osd.1   |   |   osd.2   |   |   osd.3   | 
    +-----------+   +-----------+   +-----------+   +-----------+
 
-Each node (device or bucket) in the hierarchy has a *weight*
-that indicates the relative proportion of the total
-data that device or hierarchy subtree should store.  Weights are set
-at the leaves, indicating the size of the device, and automatically
-sum up the tree, such that the weight of the ``root`` node
-will be the total of all devices contained beneath it.  Normally
-weights are in units of terabytes (TB).
 
-You can get a simple view the of CRUSH hierarchy for your cluster,
-including weights, with:
+Each node (device or bucket) in the hierarchy has a *weight* that indicates the
+relative proportion of the total data that should be stored by that device or
+hierarchy subtree. Weights are set at the leaves, indicating the size of the
+device. These weights automatically sum in an 'up the tree' direction: that is,
+the weight of the ``root`` node will be the sum of the weights of all devices
+contained under it. Weights are typically measured in terabytes (TB).
```

Comment:
I think Tebibytes strictly, though the term sticks in my craw.

Reply:
Accepted.